apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJobIT.java

```diff
@@ -59,6 +59,8 @@ public void emptyBucketTest() throws Exception {
       Assert.assertEquals(1, listener.getNoDataAvailableEvents());
       Assert.assertEquals(0, listener.getDataEvents().size());
       Assert.assertEquals(0, listener.getErrorEvents().size());
+    } catch (Exception e) {
+      LOGGER.error("EXCEPTION: %s", e.getMessage());
```

Comment:
possibly....the problem is there's a giant leap between the assertion tests and something downatream that may have throw the exception and no error message is provided....so we go from ty to finally and all we know is something went south; so the log error was there to gleen the problem.

Reply:
Printing out the message is one thing and maybe it is helpful but if this is going to stay here the exception needs to be re-thrown otherwise the test won't fail on an exception.

Reply:
yea, I thought there was a throw in that catch; am just going to remove exception handler. there; the log/printstack
 message will show up in one of the files.